_data/translations.yml

```diff
@@ -38,7 +38,7 @@ en_US:
   "La Thym": "The Thym"
   "Épisodes précédents": "Last editions"
   "Pour en savoir plus": "About"
-  "Who you gonna call?": "Who you gonna call?"
+  "On reste en contact": "Let's stay in touch"
```

Comment:
Et la trad, j'aurais mis _Shall we keep in touch?_ pour respecter la formulation sous forme de question (_Let's stay_ est plus affirmatif).